azure-pipelines.yml

```diff
@@ -24,83 +25,112 @@ stages:
-      value: s118d01-app
-
-    - name: SourceSlot
-      value: staging
-
-    - name: WebAppName
-      value: s118d01-app-as
-
-    - name : ExtraTerraformVariables
-      value:
-
-    steps:
-    - template: templates/release.yml #Run Release Template
-      parameters :
-        ServiceConnection : azdo.pipelines.cip.S118D.armfe1ef140-8bef-4043-b5ee-c449e6f951ef
-
-- stage: Review
-  condition: and(succeeded('Build'), ne(variables['Build.SourceBranch'], 'refs/heads/master'),  ne(variables['System.PullRequest.PullRequestNumber'], ''))
+  - deployment: DeployDev
+    displayName: deploy to development environment
+    pool:
+      vmImage: 'ubuntu-latest'
+    environment: 's118-teacherpaymentsservice-development'
+    strategy:
+      runOnce:
+        deploy:
+          steps:
+          - checkout: self
+          - template: templates/release.yml
+            parameters :
+              backendAzureRmContainerName : s118d01conttfstate
+              backendAzureRmKey : terraform.tfstate
+              backendAzureRmResourceGroupName : s118d01-tfbackend
+              backendAzureRmStorageAccountName : s118d01tfbackendsa
+              env : development
+              KeyVaultId : /subscriptions/8655985a-2f87-44d7-a541-0be9a8c2779d/resourceGroups/s118d01-secrets/providers/Microsoft.KeyVault/vaults/s118d01-secrets-kv
+              ResourceGroupName : s118d01-app
+              SourceSlot: staging
+              WebAppName: s118d01-app-as
+              ExtraTerraformVariables :
+              ServiceConnection : azdo.pipelines.cip.S118D.armfe1ef140-8bef-4043-b5ee-c449e6f951ef
+
+
+- stage: TestRelease
+  #condition: and(succeeded('DevelopmentRelease'), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
   jobs:
-  - job: DeployReview
-    variables:
-    - name: backendAzureRmContainerName
-      value: s118d02conttfstate
-
-    - name: backendAzureRmKey
-      value: $(app_name).tfstate
-
-    - name: backendAzureRmResourceGroupName
-      value: s118d02-review-tfbackend
-
-    - name: backendAzureRmStorageAccountName
-      value: s118d02reviewtfbackendsa
-
-    - name: env
-      value: review
-
-    - name: KeyVaultId
-      value : /subscriptions/8655985a-2f87-44d7-a541-0be9a8c2779d/resourceGroups/s118d02-secrets/providers/Microsoft.KeyVault/vaults/s118d02-secrets-kv
-
-    - name: ResourceGroupName
-      value: s118d02-app
-
-    - name: SourceSlot
-      value: staging
-
-    - name: WebAppName
-      value: s118d02-app-$(app_name)-as
-
-    - name : ExtraTerraformVariables
-      value: -var="app_name=$(app_name)"
-
-    steps:
-    - script : echo $(Build.BuildNumber), $(app_name), $(WebAppName), $(ExtraTerraformVariables), input_container_version=$(Build.BuildNumber). workspace_variables/$(env)
-    - template: templates/release.yml #Run Release Template#
-      parameters :
-        ServiceConnection : azdo.pipelines.cip.S118D.armfe1ef140-8bef-4043-b5ee-c449e6f951ef
+  - deployment: DeployTest
```

Comment:
The other stages are DevelopmentRelease, ProductionRelease...
Can we keep it TestRelease for consistency?